### ISSUE TITLE
Adicionar/remover etapa

### ISSUE THREAD

**Issue by cv** [servicosgovbr/editor-de-servicos#84]

**Comment by almeidafab** [comment]
@jeankirchner consigo inserir mais de 10 etapas.

**Comment by cv** [comment]
Não vejo razão para um limite arbitrário em 10 etapas (o comentário acima foi editado por um de nós, imagino). Duvido muito que alguém resolva fazer isso, mas podemos adicionar esse limite na #71.

**Comment by almeidafab** [comment]
concordo com vc @cv eu tb não vejo necessidade desse limite, não sei quem definiu.
